monkeys/shuffler/kubernetes-resources/client.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: apps/v1
```

Comment:
TODO: work with `helm` to use the base files in `templates` (separate client and server files maybe?) to create these. 

Plan:
1.  Create folder called `kubernetes-resources-templates` which will contain symlinks to the according files in `templates` at the root of the repo.
2. Have a step in the Makefile to fill out these templates with helm and dump their output for each monkey in `kubernetes-resources`. 
3. Then use helm to generate the rest of the files which will scoop up the contents of `kubernetes-resources`.